aksel.nav.no/website/pages/eksempler/radio-horizontal/horizontal.tsx

```diff
@@ -0,0 +1,25 @@
+import { Radio, RadioGroup, Stack } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <RadioGroup legend="Har du barn?">
+      <Stack gap="0 6" direction={{ xs: "column", sm: "row" }} wrap={false}>
+        <Radio value="1">Ja</Radio>
+        <Radio value="0">Nei</Radio>
+      </Stack>
+    </RadioGroup>
+  );
+};
+
+// EXAMPLES DO NOT INCLUDE CONTENT BELOW THIS LINE
+export default withDsExample(Example);
+
+/* Storybook story */
+export const Demo = {
+  render: Example,
+};
+
+export const args = {
+  index: 0,
```

Comment:
```suggestion
  index: 0,
  desc: "Horisontal visning av Radio kan unntaksvis være et alternativ, men som hovedregel anbefaler vi vertikal visning."
```

Reply:
Since this is a standalone example, I thought it would be better to not have a `desc` but just write everything in Sanity instead. Then it's easier to change the text later. What do you think?

Reply:
That makes sense 👍